docs/jetstream/jetstream.md

```diff
@@ -104,6 +104,16 @@ are applied to the metrics data which is then used to calculate
 [statistics](https://github.com/mozilla/jetstream/blob/main/jetstream/statistics.py).
 Statistics data is written to BigQuery and later exported to GCS as JSON. 
 
+## Tooling and metric versioning
+
+Jetstream ensures that results get computed consistently across the entire analysis duration of an experiment.
+It prevents sudden changes of how results are computed after tooling (such as [mozanalysis] which Jetstream depends on) or default metrics get updated by using the same versions as when the analysis initially started until the experiment completes.
+
+To force jetstream to use new tooling or metric versions for an experiment, there are a few options:
+* do nothing: only experiments that are launched after the new tooling release will use the most recent version of the tooling 
+* use new tooling version without rerunning: install the [jetstream command line tooling](https://pypi.org/project/mozilla-jetstream/) locally and run `jetstream rerun-skip --experiment_slug=<slug>`. This command pretends to rerun the experiment actually re-running the queries. Rerunning an experiment will always force new versions to be used for experiments. 
+* rerun experiment: re-running an experiment will always use the most recent version of the tooling on the rerun and update the last updated timestamps of the result tables
+
```

Comment:
I'd imagine this is the part DS or other experiment stakeholders would be interested in.

Reply:
Maybe this should link to the other page as a `for more info` type thing?